packages/nimble-components/src/table/specs/table-column-specs/table-column-text-field.md

```diff
@@ -0,0 +1,175 @@
+# TableColumnTextField
+
+## Overview
+
+The `nimble-table-column-text-field` is a component that defines how to render a cell for that column in a `nimble-table` as text.
+
+### Background
+
+[Table Column API](../table-columns-hld.md)
+
+[Table Spec](../README.md)
+
+[Visual Design Spec](https://xd.adobe.com/view/5b476816-dad1-4671-b20a-efe796631c72-0e14/specs/)
+
+### Non-goals
+
+-   Defining an API that supports editable text. Once we need editable text we will determine if it makes sense to modify this component or introduce a new column type.
+
+### Features
+
+-   Offers a `placeholder` attribute that allows a user to define what text to render when no value is provided by the `nimble-table` data.
+
+---
+
+## Design
+
```

Comment:
Is your goal here to follow the example of the icons where all columns are named `nimble-table-column-xxx`? I ask because I think `nimble-table-text-field-column` is more readable, but I can also support using the same rationale as we used in the icon naming where all the columns are together when the components are sorted alphabetically.

Reply:
Yes, the idea was to follow the example of the icons for discoverability. I had the same hesitancy because I agree about the readability.